src/utils/web-preset.ts

```diff
@@ -9,52 +10,49 @@ import { isUndefined } from './helper'
  */
 let online = true
 const isOnline = () => online
+const noop = () => {}
+const ADD = 'addEventListener'
 
-// For node and React Native, `window.addEventListener` doesn't exist.
-const addWindowEventListener =
-  typeof window !== 'undefined' && !isUndefined(window.addEventListener)
-    ? window.addEventListener.bind(window)
-    : null
-const addDocumentEventListener =
-  typeof document !== 'undefined'
-    ? document.addEventListener.bind(document)
-    : null
+const hasWindow = typeof window !== 'undefined'
+const hasDocument = typeof document !== 'undefined'
+
+// For node and React Native, `add/removeEventListener` doesn't exist on window.
+const onWindowEvent =
+  hasWindow && !isUndefined(window[ADD]) ? window[ADD].bind(window) : noop
+const onDocumentEvent = hasDocument ? document[ADD].bind(document) : noop
```

Comment:
I don't think we need to bind, since we are not using `this` in our callbacks. So we can make those simpler:

```js
const addWindowListener = typeof window !== 'undefined' && isUndefined(window[ADD]) ? window[ADD] : null

...

if (addWindowListener) addWindowListener('focus', cb)
```

Same for document listeners. So we can get rid of `noop` as well, make the bundle smaller.

Reply:
removed `bind` calls. but keep the others change since they make the final bundle reduced 25bytes 😌

Reply:
I don't see your push somehow 🤔

Reply:
updated